drivers/include/tmp006.h

```diff
@@ -80,6 +80,7 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include "periph/i2c.h"
+#include "board.h"
```

Comment:
This is used for `TMP006_USE_LOW_POWER`, right? It looks a bit misplaced to me, that the driver of a temperature sensor needs to know something about the board it is used on, except for it's parameters. Is it really needed to define this in board.h? I mean, the board will work without this setting and so will the driver -> this looks application-specific to me. If there really is a good reason to set `TMP006_USE_LOW_POWER` and `TMP006_USE_RAW_VALUES` in board.h then these should be handled like the other `TMP006_PARAM*` values via tmp006_params.h. But then this change should also be introduced by a seperate PR I think.